storage/rocksdb/clone/donor.cc

```diff
@@ -68,7 +68,9 @@ class [[nodiscard]] rdb_checkpoint final {
   // Returns MySQL error code
   [[nodiscard]] int init() {
     assert(!m_active);
-    const auto result = myrocks::rocksdb_create_checkpoint(m_dir.c_str());
+    m_full_dir = make_dir_full_name(
+        m_dir.c_str(), m_next_sub_id.fetch_add(1, std::memory_order_relaxed));
```

Comment:
No need to take a C string out of a C++ string to convert it to a C++ string again:
```suggestion
        m_dir, m_next_sub_id.fetch_add(1, std::memory_order_relaxed));
```

Reply:
resolved.